gnucash/gnome-utils/gnc-main-window.cpp

```diff
@@ -269,237 +291,264 @@ GNC_DEFINE_TYPE_WITH_CODE(GncMainWindow, gnc_main_window, GTK_TYPE_WINDOW,
  *  that has been installed in this window. */
 typedef struct
 {
-    /** The merge identifier for this action group.  This number
-     *  is provided by the UI manager. */
-    guint merge_id;
     /** The action group itself.  This contains all actions added
      *  by a single menu or content plugin. */
-    GtkActionGroup *action_group;
+    GSimpleActionGroup *simple_action_group; //FIXMEb added
 } MergedActionEntry;
```

Comment:
As the struct has only one member variable left, you might as well drop it completely and just just GSimpleActionGroup directly instead. Will improve readability of the code IMO.

Reply:
Yes I was wondering about that but was still unsure about whether I needed to track these merged actions at all. Will have a look.

Reply:
I have removed all to do with merged_action_hash.